<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticaMonticulos</class>
 <widget class="QWidget" name="PracticaMonticulos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushNuevaTarea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Nueva Tarea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCambiarTarea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cambiar Primera Tarea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCumplirPrimerTarea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cumplir Primer Tarea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushImprimir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir Primer Tarea</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_CrearTarea">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCrear">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditModificados">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditModifcados">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTareaMod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tarea A Modificar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetCrear">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetModificador">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelImprimir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCumplir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>356</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
